requirements.txt

```diff
@@ -21,11 +22,14 @@ pytest==2.6.4
 python-mimeparse==0.1.4
 python-subunit==1.0.0
 repoze.lru==0.6
-six==1.9.0
+requests==2.4.3
+requests-oauthlib==0.4.1
+six==1.7.3
 sure==1.2.9
 testtools==1.5.0
 transaction==1.4.3
 translationstring==1.3
+tweepy==3.2.0
```

Comment:
Are you really using tweepy or any of these other new requirements?  If not, remove them from requirements.txt